scripts/migrate_affective_backfill.py

```
@@ -0,0 +1,354 @@
+        AND mn.text != ''
+    """
+    params = []
+
+    if since:
+        since_ts = int(datetime.fromisoformat(since).timestamp() * 1000)
+        query += f" AND mn.timestamp >= ?"
+        params.append(since_ts)
+
+    if until:
+        until_ts = int(datetime.fromisoformat(until).timestamp() * 1000)
+        query += f" AND mn.timestamp <= ?"
```

Ruff (0.15.7)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)